en_US/messages.json

```
@@ -208,5 +208,9 @@
     "neverOfferTranslation": {
       "message": "Never offer translations",
       "description": "Option to prevent the translation bar from being automatically displayed"
+    },
+    "translateWith": {
+      "message": "Translate with",
+      "description":"Context menu item to send selected text in the page to be translated by Firefox Translations popup."
```

This comment is really not clear. Does the add-on have a pop-up now? Do you have a screenshot of the feature.

---

Yes. You can install the addon from here https://addons.mozilla.org/en-US/firefox/addon/firefox-translations/ and click the browserAction to see what the popup does.

---

How can that feature be available in the AMO version if the [PR](https://github.com/mozilla/firefox-translations/pull/615/files) is not merged yet?

---

They are two different features. Please refer to the linked issue for contex (i.e. translation of selected text)

---

I feel like we're running in circles. 

I've looked at the PR before writing the first comment, but I still don't understand how this feature works. That's why I asked for a screenshot, so that we can reword the comment.

You select text, you right click and pick "Translate with Firefox Translations" (the label still needs fixing btw). And then what happens? What is the "Firefox Translations popup"? I have a feeling we should just remove popup from the comment.

---

OK, I've built the add-on from that PR.
```suggestion
      "description":"Context menu item that copies the selected text in a popup dialog where it can be translated with Firefox Translations."
```